module/x/gravity/client/cli/tx.go

```diff
@@ -27,74 +24,17 @@ func GetTxCmd(storeKey string) *cobra.Command {
 	}
 
 	gravityTxCmd.AddCommand([]*cobra.Command{
-		CmdSendToEth(),
+		CmdTransfer(),
 		CmdRequestBatch(),
 		CmdDelegateKey(),
-		GetUnsafeTestingCmd(),
 	}...)
 
 	return gravityTxCmd
 }
 
-func GetUnsafeTestingCmd() *cobra.Command {
-	testingTxCmd := &cobra.Command{
-		Use:                        "unsafe_testing",
-		Short:                      "helpers for testing. not going into production",
-		DisableFlagParsing:         true,
-		SuggestionsMinimumDistance: 2,
-		RunE:                       client.ValidateCmd,
-	}
-	testingTxCmd.AddCommand([]*cobra.Command{
-		CmdUnsafeETHPrivKey(),
-		CmdUnsafeETHAddr(),
-	}...)
-
-	return testingTxCmd
-}
-
-func CmdUnsafeETHPrivKey() *cobra.Command {
-	return &cobra.Command{
-		Use:   "gen-eth-key",
-		Short: "Generate and print a new ecdsa key",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			key, err := ethCrypto.GenerateKey()
-			if err != nil {
-				return sdkerrors.Wrap(err, "can not generate key")
-			}
-			k := "0x" + hex.EncodeToString(ethCrypto.FromECDSA(key))
-			println(k)
-			return nil
-		},
-	}
-}
-
-func CmdUnsafeETHAddr() *cobra.Command {
-	return &cobra.Command{
-		Use:   "eth-address",
-		Short: "Print address for an ECDSA eth key",
-		Args:  cobra.ExactArgs(1),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			privKeyString := args[0][2:]
-			privateKey, err := ethCrypto.HexToECDSA(privKeyString)
-			if err != nil {
-				log.Fatal(err)
-			}
-			// You've got to do all this to get an Eth address from the private key
-			publicKey := privateKey.Public()
-			publicKeyECDSA, ok := publicKey.(*ecdsa.PublicKey)
-			if !ok {
-				log.Fatal("error casting public key to ECDSA")
-			}
-			ethAddress := ethCrypto.PubkeyToAddress(*publicKeyECDSA).Hex()
-			println(ethAddress)
-			return nil
-		},
-	}
-}
-
-func CmdSendToEth() *cobra.Command {
+func CmdTransfer() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "send-to-eth [eth-dest] [amount] [bridge-fee]",
+		Use:   "transfer [eth-recipient] [amount] [bridge-fee]",
```

Comment:
transfer-to-eth seems to explain what this actually does. Also begs the question, should we support commands that are not needed by validators?

Reply:
right, I think deleting them is ok

Reply:
Having a CLI for send to ETH I think is minimum